packages/core/src/state/StateSystem.js

```diff
@@ -71,10 +72,10 @@ export default class StateSystem extends System
         /**
          * Blend mode
          * @member {number}
-         * @default 17
+         * @default PIXI.BLEND_MODES.NONE
```

Comment:
@bigtimebuddy @ivanpopelyshev should be `BLEND_MODES.NONE` instead of `PIXI.` as below ?